components/automate-cli/pkg/verifyserver/services/batchcheckservice/trigger/systemresourceschecktrigger.go

```diff
@@ -1,16 +1,107 @@
 package trigger
 
-import "github.com/chef/automate/components/automate-cli/pkg/verifyserver/models"
+import (
+	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/constants"
+	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/models"
+	"github.com/sirupsen/logrus"
+)
 
-type SystemResourceCheck struct{}
+type SystemResourceCheck interface {
```

Comment:
No need for interface for each and every trigger. All trigger checks do implement ICheck only